pkgs/by-name/qh/qhexedit2/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchpatch,
+  fetchFromGitHub,
+  qt6,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qhexedit2";
+  version = "0.8.9";
+
+  src = fetchFromGitHub {
+    owner = "Simsys";
+    repo = "qhexedit2";
+    rev = "refs/tags/${finalAttrs.version}";
+  };
```

Comment:
The main reason I upgraded to unstable was so that I could apply the patch; while there are definitely some improvements in the master branch, it's probably worth it to use upstream releases. If https://github.com/Simsys/qhexedit2/pull/152 gets merged, I will ask for a new release to be created.